mcp-server/src/index.js

```
@@ -65,11 +65,47 @@ class TaskMasterMCPServer {
+			throw new Error(`Invalid MCP_TRANSPORT: ${transportType}`);
+		}
+		const port = process.env.MCP_PORT || 3000;
+
+		let serverConfig = {
 			timeout: 120000 // 2 minutes timeout (in milliseconds)
-		});
+		};
+
+		if (transportType === 'tcp') {
+			const parsedPort = parseInt(port);
+			if (isNaN(parsedPort) || parsedPort < 1 || parsedPort > 65535) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)